packages/v4/src/content/design-guidelines/components/modal/modal.md

```diff
@@ -7,111 +7,131 @@ related: ['Button']
 import CheckIcon from '@patternfly/react-icons/dist/esm/icons/check-icon';
 import TimesIcon from '@patternfly/react-icons/dist/esm/icons/times-icon';
 
-A **modal** loads and displays important information without navigating away from the current page.
+A **modal** displays important information to a user without requiring them to navigate to a new page.
 
 ## Primary elements
-<img src="./img/modal.png" alt="Modal" />
 
-1. **Backdrop**
-2. **Modal box**
-3. **Title:** Concisely convey the purpose of the modal.
-4. **Close**
-5. **Content area**
-6. **Buttons:** Enable a user to confirm, submit, or exit. Modal dialogs can include multiple action buttons, but should be limited to one primary button
+<img src="./img/modal-call-outs.png" alt="An example of a modal in context with call outs to highlight each listed element" width="756"/>
 
-### Spacing
-#### Compact
-<img src="./img/compact.png" alt="Compact" /> 
+* **Backdrop**
+* **Modal box**
+* **Headline**
+* **Close icon**
+* **Content area**
+* **Buttons** 
 
-The compact modal dialog uses a spacing of 24px. Use this for smaller modals that don’t contain much content.
+See the "[Content](#content)" section for information about writing modal copy.
 
-#### Large
-<img src="./img/large.png" alt="Large" />
+## Spacing
 
-The large modal dialog uses a spacing of 32px. Use this for larger modals that contain a lot of content (code previews, forms, tables...).
+### Compact (24px)
+
+<img src="./img/modal-compact.png" alt="A compact modal with 24px spacers" width="600" />
+
+
+Compact modals have 24px spacing and are best used for modals that contain small amounts of content. 
+
+### Large (32 px)
+
+<img src="./img/modal-large.png" alt="A large modal with 32px spacers" width="675" />
+
+Large modals have a 32px spacing to accommodate a larger content area for elements like code previews, forms, and tables.
```

Comment:
Same here, the 32px spacing doesn't apply between the header and body, where it's actually 24px